android/app/build.gradle

```diff
@@ -118,6 +120,14 @@ dependencies {
     implementation "org.webkit:android-jsc:r245459"
     devImplementation "androidx.constraintlayout:constraintlayout:2.0.0-beta2"
     devImplementation "me.dm7.barcodescanner:zxing:1.9.13"
+    androidTestImplementation("com.wix:detox:+") { transitive = true } 
+    androidTestImplementation "junit:junit:4.12"
+}
+
+configurations.all {
+    resolutionStrategy {
+        force "androidx.annotation:annotation:1.1.0"
```

Comment:
Who's the culprit for this requirement?

Reply:
Detox specifies 1.1.0, everything that Jetifier touches requires 1.0.0 somehow.

Reply:
I validated there are no breaking changes from 1.0.0 to 1.1.0. So otherwise lgtm

Reply:
Can we revert this and enforce annotations 1.0.0 on Detox? Would Detox break? 
What this line does it enforces all libraries to use annotations 1.1.0. While it might not break anything currently; we can't attest for the future.

Reply:
Apparently it only applies on version colission, so I am covered; lgtm